pokemongo_bot/step_walker.py

```diff
@@ -20,13 +20,17 @@ def __init__(self, bot, dest_lat, dest_lng):
             dest_lng
         )
 
+        if self.bot.config.walk_min > self.bot.config.walk_max:
+            self.bot.config.walk_min, self.bot.config.walk_max = self.bot.config.walk_max, self.bot.config.walk_min
+            
         self.speed = self.bot.config.walk_max - random() * (self.bot.config.walk_max - self.bot.config.walk_min)
 
         self.destLat = dest_lat
         self.destLng = dest_lng
         self.totalDist = max(1, self.dist)
 
         if self.speed == 0:
+            self.speed = uniform(3,5)
```

Comment:
I think the original idea was that if `walk` was 0, then we should teleport straight to our destination. Aka, do it in a single step. I'm not sure that is actually possible anymore with the code that is on line 26. We should make sure we fix that functionality.